src/service/pregnancyService.js

```
@@ -0,0 +1,397 @@
+      malariaDetails: formObject.malaria_current_other,
+      heartDiseasesDetails: formObject.heart_diseases_current_other,
+      kidneyDiseasesDetails: formObject.kidney_diseases_current_other,
+    };
+  }
+
+  static async getClinicalConservation() {
+    try {
+      const response = await axiosInstance.get(`/parent/pregnancy/clinical`);
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)